modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -18106,6 +18106,8 @@ unable-to-find-article-resource-for-article-with-id-x-while-gathering-its-refere
 unable-to-find-x=Unable to find {0}.
 unable-to-import-because-the-blueprint-configuration-is-invalid=Unable to import because the blueprint configuration is invalid.
 unable-to-import-because-the-element-configuration-is-invalid=Unable to import because the element configuration is invalid.
+unable-to-import-blueprint-with-the-same-external-reference-code-with-the-same-as-an-existing-blueprint=Unable to import blueprint with the same external reference code as an existing blueprint.
```

Comment:
> unable-to-import-blueprint-with-the-same-external-reference-code-with-the-same-as-an-existing-blueprint
should match message and instead be
unable-to-import-blueprint-with-the-same-external-reference-code-as-an-existing-blueprint

Reply:
And same for Element message